### ISSUE TITLE
[nginx] --hsts and --uir flags not working?

### ISSUE THREAD

**Issue by emilstahl** [certbot/certbot#3641]
Am I the only one who experience that the `--hsts` and `--uir` flags isn't adding any Strict-Transport-Security nor CSP headers?

I'm running the following and isn't seeing the headers added to any of the virtualhosts.

```
./certbot-auto --nginx --rsa-key-size 4096 --hsts --redirect --uir --staple-ocsp --must-staple -m hostmaster@domain.dk -d domain.dk -d somesub.domain.dk -d other.domain.dk
```

 The other relevant certbot stuff are however added.

```
# ./certbot-auto --version
letsencrypt 0.9.3
# cat /etc/redhat-release
CentOS Linux release 7.2.1511 (Core)
```

**Comment by bmw** [comment]
The current Nginx plugin doesn't support enabling HSTS or UIR. This isn't at all clear to the user though. We should either:

1. Check that requested enhancement isn't supported by the selected plugin early and error out with a helpful error message.
2. Print a warning saying that the requested enhancement isn't supported by the selected plugin and continue execution.

I also created #3648 and #3649.

**Comment by emilstahl** [comment]
👍 

I think 2 is great, as it is non breaking issues.

**Comment by bmw** [comment]
Fixed in #3641

**Comment by haarts** [comment]
You mean 'Fixed in https://github.com/certbot/certbot/pull/3688' ;)
